Tests/Tactics/Sym.lean

```diff
@@ -7,15 +7,15 @@ import Proofs.«AES-GCM».GCMGmultV8Sym
 
 namespace GCMGmultV8Program
 
--- Test the behaviour of `sym1_n` when `h_err` and `h_sp` are not present
+-- Test the behaviour of `sym_n` when `h_err` and `h_sp` are not present
 /-- warning: declaration uses 'sorry' -/ -- `guard_msgs` silences the warning
 #guard_msgs in example (s0 sf : ArmState)
     (h_s0_program : s0.program = gcm_gmult_v8_program)
     (h_s0_pc : read_pc s0 = gcm_gmult_v8_program.min)
     (h_run : sf = run gcm_gmult_v8_program.length s0) :
     read_err sf = .None := by
   simp (config := {ground := true}) only [Option.some.injEq] at h_s0_pc h_run
-  sym1_n 1
+  sym_n 1
   · sorry
   · exact (sorry : CheckSPAlignment s0)
   · exact (sorry : read_err s0 = .None)
```

Comment:
Wondering why the behavior is different for a missing `read_pc` hyp. vs `read_err` hyp. IIRC, in the former, we get an error and in the latter, we get a new subgoal.

Reply:
As discussed: we need the `read_pc` to be present, since we infer the value of the PC, hence, which stepi lemma to apply, from its type. 
`read_err`, OTOH, does not give us any information, we only need to know that it's true. Thus, if it's not present, we can create a new goal for it.

If we wanted to create a new goal for `read_pc`, we'd have to somehow guess the value of the PC. We *could* do that, e.g., by taking `Program.min` as a logical starting point, but this is necessarily heuristics-based, so I figured throwing an error is simpler (same thing for `h_program`, except that it'd be even harder to make any sensible guess for the right program)